### ISSUE TITLE
schema

### ISSUE THREAD

**Issue by jalbertbowden** [Code4HR/open-health-inspection-api#3]
implement schema markup roles for reviews, here's the spec:  
http://schema.org/Review  
ideally, health department would implement this when they publish the data...but we should use it anyways on all things we do that are government reviewed.  
i'm not sure if this is the correct train of thought, as i am not very up-to-speed on the apps functionality....but if we are posting the data in documents, aside from using them in the mobile search, then schema should be implemented.  

tl;dr  
we may not need this @ all; if we do, its easy peasy markup changes that i'll be more than happy to implement if called for.
